src/bindgen/processor/filter_methods.cr

```diff
@@ -1,11 +1,16 @@
 module Bindgen
   module Processor
-    # Processor removing methods that are to be ignored.
+    # Processor removing methods that are to be ignored.  Also performs fix-ups
+    # on certain methods.
     #
     # Right now, methods can be ignored by any of:
     # 1. Ignoring a type the method uses as argument or result type
     # 2. By adding the name to `type.CLASS.ignore_methods`
     # 3. Methods using anonymous types are always ignored
+    #
+    # The fix-ups include:
```

Comment:
I think this should be part of a different or new processor.  `FilterMethods` up to now really just throws stuff away if something matches, but doesn't modify a match.